iocage_lib/ioc_json.py

```diff
@@ -868,6 +868,13 @@ def check_config(self, conf, default=False):
         if not conf.get('min_dyn_devfs_ruleset'):
             conf['min_dyn_devfs_ruleset'] = '1000'
 
+        # Version 28 keys
+        for x in range(0, 4):
+            if not conf.get(f"vnet{x}_mtu"):
+                conf[f"vnet{x}_mtu"] = 'auto'
+        if not conf.get(f"vnet_default_mtu"):
```

Comment:
This does not have to be a `f-string`

Reply:
Yep, fair point. Got a bit keen on the copy/paste. I've pushed a patch for this one to only use f-strings where there is variable substitution.